src/rust/catloop/socket.rs

```diff
@@ -87,6 +92,8 @@ impl Socket {
             catmem_qd: None,
             local: None,
             remote: None,
+            backlog: 1,
```

Comment:
Does it make sense for this to default to 1? Is that the default on Linux or something?